Nos/Views/Settings/SettingsView.swift

```diff
@@ -41,220 +42,230 @@ struct SettingsView: View {
     }
 
     var body: some View {
-        Form {
-            Section {
-                HStack {
-                    Text(String(repeating: "•", count: 63))
-                        .foregroundColor(.primaryTxt)
-                        .font(.clarity(.regular, textStyle: .body))
-                        .lineLimit(1)
-                        .accessibilityLabel(Text(.localizable.privateKey))
+        /// This ZStack is necessary to make the custom delete alert view overlay the main content
+        /// to mimic Apple's default AlertView
+        ZStack {
```

Comment:
Adding a `ZStack` here works, but another option is to use the `.overlay()` modifier. It allows you to present a view on top of another view. So in this case it could look something like this:
```
Form {
   ...
}
.overlay(
   if showDeleteConfirmationAlert {
       deleteConfirmationView
   }
)
```

In this case I think it's a better option because it removes some indentation from the rest of this code (which is getting pretty long, we are probably due for a refactor here but I don't expect you to do one now).

Reply:
This is better. I wanted to do a refactor, but decided against it to prevent massive merge conflicts due to the large file changes in other PRs.

Reply:
fixed!